### DISCUSSION TITLE
Event calculator

### DISCUSSION THREAD

**Discussion by Kshyhoo67** [fedorovvl/tso_client#94] [General]
There is an event calculator on the [xxx ](https://docs.google.com/spreadsheets/d/1SkkC1UFdVhKfuTqAHgGB4Yy-FtRGnchvdl25ng2l0bM/edit#gid=938815709) site. Can it be implemented for the client? Unfortunately, I do not have such skills ;)

**Comment by fedorovvl** [comment]
for what? use site =)

**Comment by Kshyhoo67** [comment]
For convenience. This site is used by many, so the settings keep changing.
